instrumentation/opentelemetry_ecto/lib/opentelemetry_ecto.ex

```diff
@@ -125,8 +135,8 @@ defmodule OpentelemetryEcto do
 
     OpenTelemetry.Span.end_span(s)
 
-    if parent_context != :undefined do
-      OpenTelemetry.Ctx.detach(parent_context)
+    if parent_token != :undefined do
+      OpenTelemetry.Ctx.detach(parent_token)
```

Comment:
The token and context are both maps and they're both the context.

Reply:
Still better to use the name "token" and pretend it isn't a context just to not rely on it in case something were to change in the future. It probably won't... but still